tsconfig.build.json

```diff
@@ -18,6 +18,7 @@
     "./src/**/__tests__/**/*",
     "./src/**/__snapshots__/**/*",
     "./src/**/*.test.ts",
+    "./src/**/*.test-d.ts",
```

Comment:
`test-d.ts` files were included in the dist folder before. I've added it to the excludes here and in `tsup.config.ts`.

Reply:
Oops, good catch.